### ISSUE TITLE
AWS Deepracer on the spot live  stream broken ?

### ISSUE THREAD

**Issue by 1kaushik1** [aws-deepracer-community/deepracer-on-the-spot#119]
Hello,
I have initiated a training and i can access the menu.html page, but when i try to access the live stream i get the page as attached

<img width="960" alt="2024-04-09 11_31_21-DR-0 - first-model" src="https://github.com/aws-deepracer-community/deepracer-on-the-spot/assets/25524420/7a33cb9e-339b-4ec5-8bf8-dbe41d82ed59">

I tried multiple browsers, devices, etc, but its the same.
Can someone help here ?

**Comment by MarkRoss-Eviden** [comment]
@1kaushik1 I don't have the issue myself.  If you change 'Cameras' to one of the other drop downs do you see the video feeds there?  Also can you share your run.env settings as you can turn off individual video feeds there too

**Comment by A852042** [comment]
Hi @MarkRoss-Eviden , thank you for getting back, yes i did try the the different cameras and it was the same, unfortunately i don't have the env files anymore as  I cleaned up everything. 
One thing i tried differently is that i cloned the Eviden branch instead of the main branch and the issue was fixed. So i'm not really sure if there was an issue with my env files (from the main branch). But in any case, things seem to be working now.

**Comment by 1kaushik1** [comment]
Hi @MarkRoss-Eviden ,thank you for getting back, yes i did try the the different cameras and it was the same, unfortunately i don't have the env files anymore as I cleaned up everything.
One thing i tried differently is that i cloned the Eviden branch instead of the main branch and the issue was fixed. So i'm not really sure if there was an issue with my env files (from the main branch). But in any case, things seem to be working now.
